src/openvic-simulation/map/State.cpp

```diff
@@ -65,10 +66,17 @@ void State::update_gamestate() {
 	const int32_t potential_workforce_in_state = 0; // sum of worker pops, regardless of employment
 	const int32_t potential_employment_in_state = 0; // sum of (factory level * production method base_workforce_size)
 
-	industrial_power = total_factory_levels_in_state * std::clamp(
-		(fixed_point_t { potential_workforce_in_state } / 100).floor() * 400 / potential_employment_in_state,
-		fixed_point_t::_0_20(), fixed_point_t::_4()
-	);
+	fixed_point_t unclamped_score_per_factory = fixed_point_t::_0();
+
+	if(potential_employment_in_state > 0) {
+		unclamped_score_per_factory = (fixed_point_t { potential_workforce_in_state } / fixed_point_t::_100()).floor() * 400 / potential_employment_in_state;
+	}
+	else {
```

Comment:
I added spaces between if & (.
Why add a space after , at the end of line?
I can regex replace it everywhere.

Reply:
Only add spaces for `,`s that *aren't* at the end of the line. I mentioned the end of the line in case you used regex, so you don't accidentally add them after commas at the end of lines

Reply:
Right, that makes sense. I must have misread.